doc/reference/configuration/configuration_reference.rst

```diff
@@ -2439,7 +2505,11 @@ The ``process`` defines configuration parameters of the Tarantool process in the
 
 .. confval:: process.pid_file
 
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
     Store the process id in this file.
+    Can be relative to :ref:`process.work_dir <configuration_reference_process_work_dir>`.
```

Comment:
Nit: Technically speaking, the option may contain a relative file path and in this case it is interpreted as relative to `process.work_dir`.

It is a bit more precise comparing to 'can be relative to `process.work_dir`'.